core/state/statedb.go

```diff
@@ -500,9 +500,6 @@ func (s *StateDB) SelfDestruct(addr common.Address) {
 	// Regardless of whether it is already destructed or not, we do have to
 	// journal the balance-change, if we set it to zero here.
 	if !stateObject.Balance().IsZero() {
-		if s.logger != nil && s.logger.OnBalanceChange != nil {
-			s.logger.OnBalanceChange(addr, stateObject.Balance().ToBig(), new(big.Int), tracing.BalanceDecreaseSelfdestruct)
-		}
```

Comment:
How come this was removed?

Reply:
The logger will be invoked in `SetBalance`

Reply:
Right, good catch!